packages/runtime/container-runtime/src/channelCollection.ts

```diff
@@ -816,7 +816,7 @@ export class ChannelCollection implements IFluidDataStoreChannel, IDisposable {
 		message: ISequencedDocumentMessage,
 		local: boolean,
 		localMessageMetadata: unknown,
-		addedOutboundReference?: (fromNodePath: string, toNodePath: string) => void,
+		addedOutboundReference: (fromNodePath: string, toNodePath: string) => void,
```

Comment:
Update the interface too to make this required?

Reply:
I think we should remove this paramater. Data store runtime doesn't use it. It calls the `addedGCOutboundRoute` method of data store context. channel collection could also directly call `addedOutboundRoute` of the parent context.

Reply:
I removed it. Please take a look.